deliravision/torch/metrics.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+import logging
+
+from deliravision.torch.utils.tensor_ops import make_onehot_npy
+
+
+logger = logging.getLogger(__file__)
+
+
+def compute_stat_score(pred, label, n):
+    """
+    Compute metric statistics (true positive, false positive, true negative,
```

Comment:
maybe factor out the scores to own functions and call them here? Otherwise one cannot calc only one of them